MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js

```diff
@@ -75,16 +84,21 @@ define([
             var saveDetailsUrl = this.mediaGalleryEditDetails().saveDetailsUrl,
                 modalElement = $(this.modalSelector),
                 dataForm = modalElement.find('#image-edit-details-form'),
-                imageId = this.imageModel().getSelected().id;
+                imageId = this.imageModel().getSelected().id,
+                imageDetails = this.mediaGalleryImageDetails();
 
             if (dataForm.validation('isValid')) {
                 saveDetails(
                     saveDetailsUrl,
-                    dataForm
+                    modalElement.find('#image-edit-details-form')
                 ).then(function () {
                     this.closeModal();
                     this.imageModel().reloadGrid();
-                    this.mediaGalleryImageDetails().removeCached(imageId);
+                    imageDetails.removeCached(imageId);
+
+                    if ($(imageDetails.modalWindowSelector).hasClass('_show')) {
```

Comment:
It would be nice to wrap this into a function inside the `imageDetails` object, that will prevent code duplication if we need to re-use this method.